styles.css

```diff
@@ -123,3 +120,364 @@ nav {
 .fw-400 {
   font-weight: 400;
 }
+
+.works {
+  display: grid;
+  background-color: #fff;
+}
+
+.recent-works h2 {
+  margin-top: 120px;
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #1c1f19;
+}
+
+.border {
+  width: 100%;
+  color: #212121;
+  margin-top: 24px;
+  margin-bottom: 80px;
+}
+
+.recent-works img {
+  margin-bottom: 24px;
+}
+
+.recent-works h3 {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  margin-bottom: 12px;
+}
+
+.recent-works p {
+  width: 327px;
+  height: 147px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.projects {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+  align-items: center;
+  padding: 10px 12px;
+  width: auto;
+  height: 40px;
+  list-style: none;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.projects li {
+  background: #fff;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+}
+
+.projects li a {
+  color: initial;
+  text-decoration: inherit;
+  font-family: 'inter', sans-serif;
+}
+
+.orange-btn {
+  color: #fff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  width: 124px;
+  height: 48px;
+  background: #ff6f00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  border-style: none;
+  margin-bottom: 24px;
+}
+
+.project-data {
+  color: white;
+  display: grid;
+  height: 386px;
+  width: 327px;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.2) 90.94%), url(assets/Img\ Placeholder2.svg);
+}
+
+.project-data h2 {
+  margin: 84px 16px 12px 16px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.project-data p {
+  margin: 0 16px 12px 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.seeProject-btn {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  flex: none;
+  flex-grow: 0;
+  width: 327px;
+  height: 48px;
+  background: #ff6b00;
```

Comment:
You are using a few colors multiple times, for example, `#fff/white`, `#ff6b00` etc. You should try to write DRY code.
To write DRY code, try not to repeat the same colors over and over again. To deal with repetition, you can define your color variables once, and then use them again. For Example:
```  
  --color-primary: #272a31;
  --color-secondary: #ec5242;
  --color-tertiary: #d3d3d3;
  --color-white: #fff;
  --color-grey: #585858;
  --color-light-grey: #ebf0ee;
```
or like
```
  --primary: #272a31;
  --secondary: #ec5242;
  --tertiary: #d3d3d3;
```